woocommerce/woocommerce-gutenberg-products-block
# Move legacy block to “legacy” folder

Starting a deprecation process: let's move the legacy block to a `legacy` folder. This'll make it easier to see what files are "current" vs deprecated.

There should be no functionality change in the plugin. 

To test:
- `npm run build`, manually test the block
- Expect: no errors in building, no difference in using the block
- `npm test`
- Tests run and pass
